jenkins/pull_request_creator.py

```diff
@@ -126,7 +126,8 @@ def delete_old_pull_requests(self):
 
     def _delete_old_branch(self):
         LOGGER.info("Deleting existing old branch with same name")
-        self.github_helper.delete_branch(self.repository, self.branch)
+        branch_name = self.branch.split('/', 2)[2]
```

Comment:
`self.branch` contains branch name in the format `refs/heads/BRANCH-NAME` but the delete_branch function of github_helpers expect branch name in the format without `refs/heads/` as can be seen [here](https://github.com/edx/testeng-ci/blob/90c1ad86377f1fd9f37a5fedca25a6956bc2f332/jenkins/github_helpers.py#L174) and itself appends `heads/` with the branch name before [deletion](https://github.com/edx/testeng-ci/blob/90c1ad86377f1fd9f37a5fedca25a6956bc2f332/jenkins/github_helpers.py#L304)